### PR TITLE
Change ConnectionConfiguration getXmlLang method 

### PR DESCRIPTION
**Pull request by vkolesnikov-intermedia**

### CODE REVIEW

File: smack-core/src/main/java/org/jivesoftware/smack/ConnectionConfiguration.java [hunk 1]
```diff
@@ -598,13 +598,18 @@ public Locale getLanguage() {
...
+        if (language == null) return null;
+        if (language.getLanguage().isEmpty()) return "";
+        String result = language.getLanguage();
+        String country = language.getCountry();
+        if (!result.isEmpty()) {
+            if (!country.isEmpty()) {
+                result = result + "-" + country;
+            }
+        } else {
+            result = country;
```
Comment (krasnov-k) [inline smack-core/src/main/java/org/jivesoftware/smack/ConnectionConfiguration.java:610]: country can be empty here. Is it ok?

**Review by anikitin-intermedia — APPROVED** [review]

File: smack-core/src/main/java/org/jivesoftware/smack/ConnectionConfiguration.java (hunk 1 above)
Comment (vkolesnikov-intermedia) [reply smack-core/src/main/java/org/jivesoftware/smack/ConnectionConfiguration.java:610]: Yes, it other branch. But I doubted about empty language. 
Probably condition must be more simple, just:
```
            if (!country.isEmpty()) {
                result = result + "-" + country;
            }
```
I'll fix

Comment (vkolesnikov-intermedia) [reply smack-core/src/main/java/org/jivesoftware/smack/ConnectionConfiguration.java]: fxd

Comment (vkolesnikov-intermedia) [reply smack-core/src/main/java/org/jivesoftware/smack/ConnectionConfiguration.java:610]: No, getLanguage really can be empty

Comment (vkolesnikov-intermedia) [reply smack-core/src/main/java/org/jivesoftware/smack/ConnectionConfiguration.java]: Fixed, it's already handle above.

**Review by krasnov-k — APPROVED** [review]

**Pull request merged**